src/scripts/app.js

```diff
@@ -436,14 +436,19 @@ export default class InteractiveBook extends H5P.EventDispatcher {
      */
     this.on('resize', this.resize, this);
 
-    this.on('toggleMenu', () => {
+    this.on('toggleMenu', (event) => {
+      // Set nav focus flag to avoid auto-scroll in content list page
+      let focusNav = true;
+      if (event.data && event.data.shouldNotFocusNav) {
```

Comment:
Sorry, don't want to be a smart-ass, but what about simply using
```
const focusNav = !event.data?.shouldNotFocusNav;
```
Or not even use an extra variable at all and use 
```
if (this.pageContent.sidebarIsOpen && !event.data?.shouldNotFocusNav)`
```
later on?

And what about a positive variable name `shouldFocusNav` to avoid double negation?

Then you'd only need `this.trigger('toggleMenu', { shouldFocusNav: false });` and `if (this.pageContent.sidebarIsOpen && event.data?.shouldFocusNav)` (unless `shouldNotFocusNav` was used before somewhere else, did not check that)

Reply:
Code has been improved! Thanks for the suggestion.

Reply:
@ravimajithia You're welcome. Sorry for dropping in that way without being asked for a review.